src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+Transactions also have one additional property not included in spans, called `transaction_name`, which is used in the UI to identify the transaction. Common examples of `transaction_name` values include endpoint paths (like `/store/checkout/` or `api/v2/users/&lt;user_id&gt;/`) for backend request transactions, task names (like `data.cleanup.delete_inactive_users`) for cron job transactions, and URLs (like `https://docs.sentry.io/performance/distributed-tracing/`) for page-load transactions.
 
-## Transactions
+**Spans**
 
-[{% asset performance/anatomy-of-transaction.png alt="Diagram illustrating how a transaction is composed of many spans." %}]({% asset performance/anatomy-of-transaction.png @path %})
+The majority of the data in a transaction resides in the individual spans the transaction contains. Span data includes:
 
-Traces in Sentry are segmented into [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) called [**transactions**]({%- link _documentation/performance/performance-glossary.md -%}#transaction). When instrumenting the application with tracing, collected spans are grouped within an encompassing top-level span called the **transaction span**. This notion of a transaction is specific to Sentry.
+- `parent_span_id`: ties the span to its parent span
+- `op`: short string identifying the type of operation the span is measuring
+- `start_timestamp`: when the span was opened
+- `end_timestamp`: when the span was closed
+- `description`: longer description of the span's operation, often specific to that instance (optional)
+- `status`: short code indicating operation's status (optional)
+- `tags`: key-value pairs holding additional data about the span (optional)
+- `data`: arbitrarily-structured additional data about the span (optional)
 
-If you are collecting transactions from more than a single machine, you will likely encounter [**clock skew**]({%- link _documentation/performance/performance-glossary.md -%}#clock-skew).
+An example use of the `op` and `description` properties together is `op: sql.query` and `description: SELECT * FROM users WHERE last_active < DATE_SUB(CURRENT_DATE, INTERVAL 1 YEAR)`. The `status` property is often used to indicate the success or failure of the span's operation, or for a response code in the case of HTTP requests. Finally, `tags` and `data` allow you attach further contextual information to the span, such as `function: middleware.auth.is_authenticated` for a function call or `request: {url: ..., headers: ... , body: ...}` for an HTTP request.
```

Comment:
This example is fine as is, but I think if would be useful to highlight that the description should be consistent across instances.
I changed the description to have a placeholder, and added an example of where to put the instance specific value in tags.

```suggestion
An example use of the `op` and `description` properties together is `op: sql.query` and `description: SELECT * FROM users WHERE id = ?`. The `status` property is often used to indicate the success or failure of the span's operation, or for a response code in the case of HTTP requests. Finally, `tags` and `data` allow you attach further contextual information to the span, such as `user_id:  1123` in the example above,  `function: middleware.auth.is_authenticated` for a function call or `request: {url: ..., headers: ... , body: ...}` for an HTTP request.
```

Reply:
See my question above. I admit that here, specifically with DB queries, we are doing exactly what you say (holy cow, that's a lot of placeholders!): 
![image](https://user-images.githubusercontent.com/14812505/81578186-4e1e3480-935f-11ea-943c-ccb73126a745.png)

So in general I want to describe whatever the best practice is accurately, but first I want to be sure I'm clear on what it is!

Reply:
I'll just answer your question here: Yes we are talking about the same thing when we talk about "consistent across instances". Your example of the transaction name is a good one. You can have many traces of the same transaction, but the transaction name will always be the same. The span description should work the same way. So in your example above, with the `s1.sentry-cdn.com` link, that is a BAD example. The idea is that if I open two different traces in two different browser tabs, the op/description of the spans should be the same.

Reply:
Okay, noted.  (I'll let _you_ go tell that to whoever sat that up, LOL.)

Took your edit of the `description` description. I kept the overall idea of my query (just because it calls back to one of the examples above), but genericized it to `description: SELECT * FROM users WHERE last_active < %s`. LMK if that still sounds off to you.